addons/im_livechat/static/src/core/thread_service_patch.js

```diff
@@ -63,7 +63,38 @@ patch(ThreadService.prototype, "im_livechat", {
         this.store.discuss.livechat.threads.sort((localId_1, localId_2) => {
             const thread1 = this.store.threads[localId_1];
             const thread2 = this.store.threads[localId_2];
-            return thread2.lastInterestDateTime.ts - thread1.lastInterestDateTime.ts;
+            return thread2.lastInterestDateTime?.ts - thread1.lastInterestDateTime?.ts;
         });
     },
+
+    /**
+     * @returns {boolean} Whether the livechat thread changed.
```

Comment:
This can return `true` or `undefined`. Maybe just return `false` instead of undefined?

Reply:
done :+1: